src/services/index.ts

```diff
@@ -9,7 +9,7 @@ import {  PalletStakingIndividualExposure } from '@polkadot/types/lookup'
 export async function nominatorThreshold(api: ApiPromise) {
 	const DOT = 10000000000;
 	const t = new BN(DOT).mul(new BN(140));
-	const np = (await api.query.staking.nominators.entries()).map(async ([sk, _]) => {
+	const np = (await api.query.staking.nominators.entries()).map(async ([sk]) => {
```

Comment:
do we have a syntax like `[sk, ...]` perhaps? this looks a bit strange to me. For example, let's say you want to bind a variable to the first item of a list. Then I'd expect either `let [x, _] = list` or `let [x, ..] = list`. But seems like your changes suggest that `let [x] = list` actually is equal to `let x = list[0]`. Am I correct?

Reply:
Yes. Javascript will 'read' the first one given (afaik)

Reply:
I learned something new then! thanks